src/main/routes/routes.ts

```diff
@@ -68,6 +68,8 @@ export default function(app: Application): void {
   app.get('/single-justice-procedure', app.locals.container.cradle.singleJusticeProcedureController.get);
   app.get('/view-option', app.locals.container.cradle.viewOptionController.get);
   app.post('/view-option', app.locals.container.cradle.viewOptionController.post);
+  app.get('/sign-in', app.locals.container.cradle.signInController.get);
```

Comment:
current routes are alphabetically ordered, can we stick to that please

Reply:
done